File: src/main/java/com/webproject/service/PayexpertService.java
package com.webproject.service;

import com.webproject.entity.Payexpert;
import com.webproject.util.PageResult;
import com.webproject.util.Result;

public interface PayexpertService {

	Result add(Payexpert payexpert);

	PageResult list(Payexpert search, int page, int rows);

	Result deleteOne(String id);

}

File: src/main/webapp/js/service/payexpertService.js
//服务层
app.service('payexpertService',function($http){
	var requestUrl  =  localStorage.getItem('requestUrl');
	//增加 
	this.add=function(entity){
		return  $http.post(requestUrl +  '/payexpert/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post(requestUrl+ '/payexpert/update',entity );
	}
	
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post(requestUrl+ '/payexpert/list?page='+page+"&rows="+rows, searchEntity);
	} 
	
	//查询实体
	this.findOne=function(id){
		return $http.get(requestUrl +  '/payexpert/findOne?id='+id);
	}
	
	//查询实体
	this.deleteOne=function(id){
		return $http.get(requestUrl +  '/payexpert/deleteOne?id='+id);
	}
	
});
